### DISCUSSION TITLE
Hi I'd like to discuss new project with you.

### DISCUSSION THREAD

**Discussion by agiledev624** [lishaduck/lishaduck#67] [General]
Please let me know your best messenger handle. Thanks

**Comment by lishaduck** [comment]
> Please let me know your best messenger handle. Thanks

@lishaduck on github.com :)
I see no reason why this sort of discussion need not be held in the open. Transparency is important to me, so I neither have nor do I desire any forms of private messaging.

**Reply by agiledev624** [reply]
I was going to discuss my own project with you for detailed scope. It would be transparent as well.
I'm not sure if you're interested in that discussion.

**Reply by lishaduck** [reply]
You've got me interested, so please feel free to detail it.

**Reply by agiledev624** [reply]
Please let me know your email or messenger to go through details. Thanks.

**Reply by lishaduck** [reply]
Is there a reason you can't send this information here? I don't intend on disclosing that information here. For all I know, you could be just trying to get it to turn around and sell it.

**Reply by agiledev624** [reply]
If you let me to get involved into detailed chat, you will get more interested. Thanks.

**Reply by lishaduck** [reply]
> If you let me to get involved into detailed chat, you will get more interested. Thanks.

Do you understand how suspicious that sounds?
If you want to chat in detail, this is already a format that allows for depth over breadth.

**Reply by agiledev624** [reply]
Ok, my tg handle is @ agiledev0624, I'd like to chat with you. Regards!

**Reply by lishaduck** [reply]
Is "tg" telegram? I don't use telegram.
Again, if you want to chat, I see no reason not to do it here. My regards!
